dcat/index.html

```diff
@@ -1193,9 +1193,10 @@ <h4>Property: identifier</h4>
             <table class="definition">
                 <thead><tr><th>RDF Property:</th><th><a href="http://purl.org/dc/terms/identifier">dcterms:identifier</a></th></tr></thead>
                 <tbody>
-                <tr><td class="prop">Definition:</td><td>A unique identifier of the item.</td></tr>
-                <tr><td class="prop">Range:</td><td><a href="http://www.w3.org/2000/01/rdf-schema#Literal"><code>rdfs:Literal</code></a></td></tr>
-                <tr><td class="prop">Usage note:</td><td>The identifier might be used as part of the URI of the item, but still having it represented explicitly is useful.</td></tr>
+                <tr><td class="prop">Definition:</td><td>A unique identifier of the resource being described or catalogued </td></tr>
+                <tr><td class="prop">Range:</td><td><a href="http://www.w3.org/2000/01/rdf-schema#Literal"><code>rdfs:Literal</code></a> usually a </td></tr>
+                <tr><td class="prop">Usage note:</td><td>The identifier is a text string which is assigned to the resource to provide an unambiguous reference within a particular context. A resource may have more than one identifier assigned by different authorities, or for use in different contexts.</td></tr>
```

Comment:
```suggestion
                <tr><td class="prop">Usage note:</td><td>The identifier is a text string which is assigned to the resource to provide an unambiguous reference within a particular context.</td></tr>
```

Reply:
As reported in https://github.com/w3c/dxwg/issues/771#issuecomment-798905066 , the use of `dct:identifier` for multiple identifiers is in conflict with the guidelines in https://www.w3.org/TR/vocab-dcat-2/#dereferenceable-identifiers